cmd/cli_test.go

```diff
@@ -80,40 +80,29 @@ func TestCli(t *testing.T) {
 			t.Errorf("There should be an error log, instead got: '%s'", outStr)
 		}
 
-		out, err = ioutil.ReadFile(outputFileName)
+		out, err = os.ReadFile(outputFileName)
 		if err != nil {
 			t.Fatal(err)
 		}
 
 		expected := `<index>
- <timestamp></timestamp>
+ <vendor>testing_vendor_index</vendor>
+ <url>test-continue-despite-errors.xml</url>
+ <>
```

Comment:
This looks wrong. I would have expected to see a timestamp but not `<>`. Is that valid xml?